### PR TITLE
added 13-erc20-staking folder

### PR DESCRIPTION
**Pull request by rn-santos227**

### CODE REVIEW

File: 13-erc20-staking/scripts/mint.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { expect } = require('chai');
+
+async function main() {
+  const [owner, addr1] = await ethers.getSigners();
+  const initialSupply = '100000000000000000000';
+  const myTokenContract = await await hre.ethers.deployContract("MyToken", [initialSupply, owner.address]);
+  await myTokenContract.waitForDeployment();
+  await myTokenContract.connect(owner).mint(addr1.address, 1000);
```
Comment (kimerran) [inline 13-erc20-staking/scripts/mint.js]: try to mint amount at least equivalent to 1 ETH or more

File: 13-erc20-staking/scripts/stake.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const { expect } = require('chai');
+
+async function main() {
+  const [owner, addr1] = await ethers.getSigners();
+  const initialSupply = '100000000000000000000';
+  const myTokenContract = await await hre.ethers.deployContract("MyToken", [initialSupply, owner.address]);
+  await myTokenContract.waitForDeployment();
+
+  await myTokenContract.connect(owner).mint(addr1.address, 1000);
```
Comment (kimerran) [inline 13-erc20-staking/scripts/stake.js]: try to mint amount at least equivalent to 1 ETH or more

**Comment by kimerran** [comment]
@rn-santos227 would love to see sample transactions as well. thanks!

File: 13-erc20-staking/scripts/mint.js (hunk 1 above)
Comment (rn-santos227) [reply 13-erc20-staking/scripts/mint.js]: is it okay to use gwei instead to save some eth.

**Pull request closed without merging**